go.mod

```diff
@@ -317,3 +317,4 @@ require (
 )
 
 replace gopkg.in/yaml.v2 => github.com/juju/yaml/v2 v2.0.0
+replace github.com/canonical/sqlair => ../../SQLair/sqlair
```

Comment:
Fix me!

Reply:
All fixed now, the bugfix I needed is in SQLair